java/src/jmri/jmrix/can/cbus/CbusBundle.properties

```diff
@@ -86,7 +86,7 @@ TrafficDirection        = Direction
 TrafficDirectionTip     = Shows packet direction > > with arrows > >
 ButtonClearLogTip       = Clear screen, does not affect logging to a file.
 ButtonStartLogTipExtra  = Will overwrite an existing file.
-NoOpenLogFile           = <html>Unable to open your console log file. <br>It may not exist if you have not yet written to it.</html>
+NoOpenLogFile           = <html><p>Unable to open your console log file. <br>It may not exist if you have not yet written to it.</p></html>
```

Comment:
What's the purpose of `<p>` here? I'm not against it, but I have never used it in cases like this.